source/mesh_refinement/isotherms.cc

```diff
@@ -0,0 +1,373 @@
+                                              "isotherms'. The first parameter takes the "
+                                              "number of the compositional field and excludes "
+                                              "it from this plugin. "
+                                              "The second parameters takes a list of isotherm "
+                                              "parameters for one isotherm separated by a ';'. "
+                                              "Each line of isotherm parameters contains four "
+                                              "subparameters. The first subparameter sets "
+                                              "the minimum refinement level for this isotherm, "
+                                              "the second subparameter sets the maximum "
+                                              "refinement level for this isotherm, the third "
+                                              "subparameter sets the minimum temperature for "
+                                              "this isotherm and the fourth subparameter sets "
+                                              "the maximum temperature for the isotherm. "
+                                              "The minimum and maximum level can be indicated "
+                                              "by the absolute number of the refinement level "
+                                              "or by using the key words 'min' or 'max', "
+                                              "corresponding respectively to the set minimum "
+                                              "and maximum refinement level. The keywords "
+                                              "'min' and 'max' may also be used in combination "
+                                              "with addition or substraction, e.g. 'max-1' "
+                                              "or 'min+1'. For formatting it is useful to "
+                                              "make use of the built-in option of ASPECT "
+                                              "to split parameters over several lines by "
+                                              "putting a backslash at the end of the line. "
+                                              "\n\n"
+                                              "Example input format of List of isotherms:\n"
+                                              "\\begin{lstlisting}"
+                                              "set List of isotherms = "
+                                              "max,\t max,\t 0,\t 1000; \\ \n\t\t 5,\t max-1,\t"
+                                              " 1000,\t 1500; \\ \n\t\t min,\t max-1,\t 1500,\t"
+                                              " 2000"
+                                              "\\end{lstlisting}"
+                                              "The criterion is checked for every quaderature point "
```

Comment:
this seems excessive. don't you think it would be enough to do this only at the midpoint of the cell?